ray-operator/controllers/ray/common/constant.go

```diff
@@ -88,7 +89,8 @@ const (
 	RAYCLUSTER_DEFAULT_REQUEUE_SECONDS      = 300
 
 	// Ray core default configurations
-	DefaultRedisPassword = "5241590000000000"
+	DefaultRedisPassword                 = "5241590000000000"
+	DefaultWorkerRayGcsReconnectTimeoutS = "600"
```

Comment:
Here, or somewhere else in the code, can we add a brief comment explaining why we picked this value? It's explained well in the PR description but it's helpful to have it in a code comment

Reply:
Added comment in https://github.com/ray-project/kuberay/pull/1036/commits/d0991ff50ac8437ed24517c4fa9c797add3349ca